code/modules/clothing/gloves/color.dm

```diff
@@ -130,21 +130,39 @@
 
 /obj/item/clothing/gloves/color/latex
 	name = "latex gloves"
-	desc = "Cheap sterile gloves made from latex."
+	desc = "Cheap sterile gloves made from latex. Transfers minor paramedic knowledge to the user via budget nanochips."
 	icon_state = "latex"
 	item_state = "lgloves"
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
 	transfer_prints = TRUE
 	resistance_flags = NONE
 
+/obj/item/clothing/gloves/color/latex/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && slot == SLOT_GLOVES)
+		ADD_TRAIT(user, TRAIT_QUICK_CARRY, CLOTHING_TRAIT)
+
+/obj/item/clothing/gloves/color/latex/dropped(mob/user)
+	. = ..()
+	REMOVE_TRAIT(user, TRAIT_QUICK_CARRY, CLOTHING_TRAIT)
+
 /obj/item/clothing/gloves/color/latex/nitrile
 	name = "nitrile gloves"
-	desc = "Pricy sterile gloves that are stronger than latex."
+	desc = "Pricy sterile gloves that are thicker than latex. Transfers intimate paramedic knowledge into the user via nanochips."
 	icon_state = "nitrile"
 	item_state = "nitrilegloves"
 	transfer_prints = FALSE
 
+/obj/item/clothing/gloves/color/latex/nitrile/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && slot == SLOT_GLOVES)
+		ADD_TRAIT(user, TRAIT_QUICKER_CARRY, CLOTHING_TRAIT)

```

Comment:
could made the trait added a var on `/obj/item/clothing/gloves/color/latex`, which would save you from having this whole block of copypaste

Reply:
IMO they should be the same trait with different sources to differentiate between quick and quicker, but this would apply as well.